README.md

```diff
@@ -58,7 +58,59 @@ facepaint(selectors: Array<Selector>) : DynamicStyleFunction
   )
   ```
   - literal `boolean` (Default: `false`) - force "slot"
```

Comment:
the original question on slack was about `literal`, could it be explained better?

Reply:
😅 whoops, I misread that. I'll update that part as well.